lib/sidekiq_workers.rb

```diff
@@ -6,6 +6,12 @@
 require 'sidekiq'
 require 'sidekiq-cron'
 
+SIDEKIQ_WORKER_BASE = Pathname.new(__FILE__).realpath.parent.parent
+$: << SIDEKIQ_WORKER_BASE # for config initializers
+$: << SIDEKIQ_WORKER_BASE + 'lib'
```

Comment:
can't we just reference relative paths rather than messing with `.parent.parent` and `$:`?

Reply:
With very pleasure. Will do the same for all other uses of $: in a separate commit